content/en/examples/pods/pod-configmap-envFrom.yaml

```
@@ -1,13 +1,14 @@
 apiVersion: v1
 kind: Pod
 metadata:
-  name: dapi-test-pod
+  name: my-pod-with-env-configmap
+  labels:
+    name: use-configmap-env
 spec:
   containers:
-    - name: test-container
-      image: k8s.gcr.io/busybox
-      command: [ "/bin/sh", "-c", "env" ]
-      envFrom:
-      - configMapRef:
-          name: special-config
-  restartPolicy: Never
+  - envFrom:
+    -configMapRef:
```

```suggestion
    - configMapRef:
```
would be more idiomatic